docs/install.md

```diff
@@ -0,0 +1,33 @@
+---
+# Discussion Forum
+
+## Install Guide
+---
+ ### Server Installation
+- Download the latest version of WAMP/XAMPP server.
+- Install it through the wizard.
+
+ ### Files Setup
+-  open C:\ > wamp > www
```

Comment:
Use single line code block syntax. Like: Open `/wamp/www`. Aside from that, saying open your web-server root is fine for most purposes, I think.